cmd/client_test.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+// Client_test tests the client.go file to ensure that the gRPC client for connecting
+// to Cloudprober works as expected. This client is used for adding and removing probes.
+
+package cmd
+
+import (
+	"github.com/golang/glog"
+	"os"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/cloudprober/examples/extensions/myprober/myprobe"
+	probes_configpb "github.com/google/cloudprober/probes/proto"
+	targetspb "github.com/google/cloudprober/targets/proto"
+	"github.com/google/cloudprober"
+	"github.com/google/cloudprober/web"
+
+	"github.com/googleinterns/step224-2020/hermes"
+)
+
+const (
+	cpCfg     string = "grpc_port:9314" // Cloudprober config
+	rpcServer string = "localhost:9314" // Cloudprober gRPC server address
+
+	testProbe0 string = "testExtension0"
```

Comment:
These should be closer to where they're used. Ideally, look at [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests). You probably want to use Table DrivenTests everywhere in go (but you don't need to implement them before we merge this :-))

Reply:
I have refactored this into a table driven test. That's a cool link and awesome approach, thanks for sharing 😄